### DISCUSSION TITLE
How to change drawer-overlay bg-opacity to be fully transparent

### DISCUSSION THREAD

**Discussion by theaspirational** [saadeghi/daisyui#1057] [Q&A] [answered]
![Screenshot_3](https://user-images.githubusercontent.com/48733736/186671664-ebccc712-f26c-47b1-9175-2a264139508b.png)

How to change drawer-overlay bg-opacity to be fully transparent ie. make overlay not visible on drawer-content?

**Comment by saadeghi** [comment, marked as answer]
`drawer-overlay` only gets opacity value when checkbox is checked so we can have a good fade effect.  
to change the background opacity value you can use this crazy class name:
```html
<input id="my-drawer" type="checkbox" class="drawer-toggle [&:checked~.drawer-side>.drawer-overlay]:[--tw-bg-opacity:0]" />
```
https://play.tailwindcss.com/HveFnBMW82

Or you can change the CSS yourself:
```css
.drawer-toggle:checked ~ .drawer-side > .drawer-overlay{
  --tw-bg-opacity: 0;
}
```
https://play.tailwindcss.com/BehenwI65b?file=css
